pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, pandoc
+, esbuild
+, deno
+, fetchurl
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${version}/quarto-${version}-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
```

Comment:
```suggestion
  version = "1.1.189";
  src = fetchurl {
    url = "https://github.com/quarto-dev/${pname}/releases/download/v${version}/quarto-${version}-linux-amd64.tar.gz";;
    sha256 = "1a3xsgqdccm4ky1xjnin1idpp8gsansskq37c00mrxz1raxn1mi7";
  };
```

Reply:
That new line is fine and substituting pname everywhere is not an improvement.

Reply:
Also, the repo is `quarto-cli` and not `quarto` so we would also have to change the package name.

Reply:
Well, that is the name of the repo not of the software.